src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -85,7 +109,7 @@ protected override void BeginProcessing()
                 {
                     try
                     {
-                        _jschema = JsonSchema.FromJsonAsync(Schema).Result;
+                        _jschema = JsonSchema.FromText(Schema);
                     }
                     catch (AggregateException ae)
```

Comment:
Now it is not async and no AggregateException is. 

I'd expect your API throw there is a problem with the schema loading/parsing.

Reply:
Yes, it'd be the JsonException mentioned before. 👍

Reply:
Interestingly, the `catch` on [line 127](https://github.com/PowerShell/PowerShell/pull/18023/files#diff-75e882f1df5a078731c96023ef526ca48672b2d6d854008d1cc9d2182e44f990L127) (catching an invalid schema, e.g. a boolean in `maximum`) would also throw a `JsonException` because the data is validated during deserialization.  I don't think this `catch` would ever be hit.